src/components/Layout.tsx

```diff
@@ -7,11 +7,11 @@ const { Content } = AntLayout;
 import Navbar from "./Navbar";
 import "../style/index.sass";
 import useSiteMetadata from "./SiteMetadata";
-import { container } from "../style/layout.module.css";
+const { container } = require("../style/layout.module.css");
```

Comment:
what is it that makes this change necessary?

Reply:
Gatsby, or at least the version I'm using, doesn't have typescript support for modules. So it wants you to make a d.ts file for each module you make. which seemed silly so I did this